kubernetes/EKS/README.md

```diff
@@ -361,78 +396,38 @@
   Succesfully created EFS & Mountpoints
   ```
 
-  
-
   Upon completion of the script it would emit a EFS volume DNS Name similar to `fs-ab1cd.efs.us-west-2.amazonaws.com` where `fs-ab1cd` is the EFS filesystem id.
 
-  
-
   You should be able to see a Security Group in your AWS Console with Inbound Rules to a NFS (Port 2049)
 
-  
-
   ![security_group](images/security_group.png)
 
-  
-
   You should also find Mount Points in your EFS console for every region where there is a Node in the Node Group.
 
-  
 
   ![](images/efs_mount.png)
 
-  
-
-  
-
-  **Prepare PersistentVolume for Deployment**
+  Create a storage class file with below specs.
 
-  We use the [ELF Provisioner Helm Chart](https://github.com/helm/charts/tree/master/stable/efs-provisioner) to create a PersistentVolume backed by EFS. Run the following command to set this up.
```

Comment:
Please add a comment that due to deprecation of the AWS EFS-Provisioner, steps have been copied over from https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/examples/kubernetes/dynamic_provisioning/README.md

For details of what these parameters mean one can reference the old example.